app-admin/vaultwarden/files/vaultwarden-envfile-1.33.2.patch

```diff
@@ -0,0 +1,63 @@
+As upstream defaults create directories where binary is executed (/usr/bin/), this patch is needed to change paths to Gentoo specific locations
+--- a/.env.template
++++ b/.env.template
+@@ -5,33 +5,29 @@
+ ## Be aware that most of these settings will be overridden if they were changed
+ ## in the admin interface. Those overrides are stored within DATA_FOLDER/config.json .
+ ##
+-## By default, Vaultwarden expects for this file to be named ".env" and located
+-## in the current working directory. If this is not the case, the environment
+-## variable ENV_FILE can be set to the location of this file prior to starting
+-## Vaultwarden.
+ 
+ ####################
+ ### Data folders ###
+ ####################
+ 
+ ## Main data folder
+-# DATA_FOLDER=data
++DATA_FOLDER=/var/lib/vaultwarden
+ 
+ ## Individual folders, these override %DATA_FOLDER%
+-# RSA_KEY_FILENAME=data/rsa_key
+-# ICON_CACHE_FOLDER=data/icon_cache
+-# ATTACHMENTS_FOLDER=data/attachments
+-# SENDS_FOLDER=data/sends
+-# TMP_FOLDER=data/tmp
++# RSA_KEY_FILENAME=/var/lib/vaultwarden/rsa_key
++ICON_CACHE_FOLDER=/var/cache/vaultwarden/icon_cache
++# ATTACHMENTS_FOLDER=/var/lib/vaultwarden/attachments
++# SENDS_FOLDER=/var/lib/vaultwarden/sends
++TMP_FOLDER=/tmp/vaultwarden
+ 
+ ## Templates data folder, by default uses embedded templates
+ ## Check source code to see the format
+-# TEMPLATES_FOLDER=data/templates
++# TEMPLATES_FOLDER=/var/lib/vaultwarden/templates
+ ## Automatically reload the templates for every request, slow, use only for development
+ # RELOAD_TEMPLATES=false
+ 
+ ## Web vault settings
+-# WEB_VAULT_FOLDER=web-vault/
++WEB_VAULT_FOLDER=/usr/share/webapps/vaultwarden-web
+ # WEB_VAULT_ENABLED=true
```

Comment:
Feels to me like in reality, vaultwarden-web should set this piece of the config, so maybe the config needs to be ripped apart a bit.

Reply:
and if this were shell, it'd be as simple as `[[ -e vaultwarden-web.conf ]] && source vaultwarden-web.conf` or something in that order